service/src/test/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java

```diff
@@ -99,6 +99,29 @@ public static HearingBooking createHearingBooking(LocalDateTime startDate, Local
             .build();
     }
 
+    public static HearingBooking createHearingBooking(LocalDateTime startDate,
+                                                      LocalDateTime endDate,
+                                                      HearingType hearingType,
+                                                      UUID cmoID) {
+        return HearingBooking.builder()
+            .type(hearingType)
+            .startDate(startDate)
+            .venue("Venue")
+            .endDate(endDate)
+            .caseManagementOrderId(cmoID)
+            .judgeAndLegalAdvisor(JudgeAndLegalAdvisor.builder()
+                .judgeTitle(HER_HONOUR_JUDGE)
+                .judgeLastName("Law")
+                .legalAdvisorName("Peter Parker")
+                .build())
+            .noticeOfHearing(DocumentReference.builder()
+                .filename("fileName")
+                .binaryUrl("binary_url")
+                .url("www.url.com")
+                .build())
+            .build();
```

Comment:
```suggestion
               return createHearingBooking(startDate, endDate).toBuilder()
            .type(hearingType)
            .caseManagementOrderId(cmoID)
            .build();
```

Could you try this please? I tried a test and it seems to work

Reply:
Nice suggestion. Changed.